tests/executor/test_dismod_runner.py

```diff
@@ -45,7 +45,7 @@ def test_dmchat_low_priority(dmchat, caplog):
     assert err == "".join(["err" + os.linesep] * 2)
 
 
-def test_dmchat_nonzero(dmchat):
+def test_dmchat_nonzero(dmchat, signals):
```

Comment:
I like this pattern.

Reply:
It's one of a few possible patterns. It's working for me so far. I think there may be a third flag, for "dismod is present on this machine".